index.html

```diff
@@ -7,51 +7,551 @@
+                    <g class="style-scope yt-icon">
+                        <path d="M12.04 4C17 4 21 7.7 21 12.3c0 4.57-4 8.28-8.96 8.28-1.76 0-3.4-.47-4.8-1.28-.15-.1-.35-.1-.52-.04-1.58.68-2.8.88-3.45.93-.2 0-.34-.26-.22-.4.6-.9 1.08-1.8 1.43-2.5.1-.2.1-.46-.04-.65C3.58 15.4 3.1 13.9 3.1 12.3c0-4.6 4-8.3 8.94-8.3zm1.6 9.56v1.58c0 .17.22.25.34.13l3.43-3.34c.1-.07.1-.2 0-.27L14 8.32c-.12-.12-.33-.04-.33.13V10c0 .1-.07.18-.17.2-3.38.52-5.04 2.73-5.74 5.1-.06.2.2.34.32.2 1.3-1.42 3.05-2.1 5.4-2.13.1 0 .2.1.2.2z"
+                            class="style-scope yt-icon"></path>
+                    </g>
+                </svg>
+            </a>
+            <a class="right-icon">
+                <svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 24 24">
+                    <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm0 3c1.66 0 3 1.34 3 3s-1.34 3-3 3-3-1.34-3-3 1.34-3 3-3zm0 14.2c-2.5 0-4.71-1.28-6-3.22.03-1.99 4-3.08 6-3.08 1.99 0 5.97 1.09 6 3.08-1.29 1.94-3.5 3.22-6 3.22z" />
+                    <path d="M0 0h24v24H0z" fill="none" /></svg>
+
+            </a>
+        </div>
+
     </header>
-    <div class="video-section">
-        <section>
-            <div class="container">
-                <span>
-                    <iframe width="1020" height="600" src="https://www.youtube.com/embed/hVSpac8wx3I" frameborder="0"
-                        allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-                    <h2>AVENGERS 4 ENDGAME Trailer (2019)</h2>
-                </span>
+    <div>
+        <div class="main-container">
+            <!-- Video Player -->
+            <div class="videoflex">
+                <div class="video-section">
+                    <section>
+
+
+                        <iframe width="750" height="400" src="https://www.youtube.com/embed/hVSpac8wx3I" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+                            allowfullscreen></iframe>
+                        <h2>Trailer Avengers 4!</h2>
+                        <div>
+                            <h3>First Trailer for AVENGERS 4 ENDGAME</h3>
+                        </div>
+
+                        <div>
+                            <h3 class="comments-title">8 Comments:&emsp;<svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false" class="style-scope yt-icon" style="pointer-events: none; width: 24px; height: 24px;"><g class="style-scope yt-icon">
+                                <path d="M3 18h6v-2H3v2zM3 6v2h18V6H3zm0 7h12v-2H3v2z" class="style-scope yt-icon"> </path>
+                                <path d="M0 0h24v24H0z" fill="none" class="style-scope yt-icon"></path>
+                              </g></svg><input type="text" value="  SORT BY"></h3>
+                              &nbsp;&nbsp;&nbsp;&nbsp;
+                              <div class="horizontal-line"></div>
```

Comment:
Just an idea:
CSS has things like the ['first of type'](https://www.w3schools.com/cssref/sel_first-of-type.asp) selector. You can do something like add a border just to the first element of a certain type/class

Reply:
Thank you we will definitely use it in the future :D

Reply:
👍